cmd/util/ledger/migrations/account_based_migration.go

```diff
@@ -275,14 +288,20 @@ func MigrateGroupConcurrently(
 		),
 	)
 
-	durations := newMigrationDurations(logTopNDurations)
+	topDurations := util.NewTopN[migrationDuration](
+		logTopNDurations,
+		func(duration migrationDuration, duration2 migrationDuration) bool {
+			return duration.Duration > duration2.Duration
```

Comment:
`heap.Pop` should return the minimum duration.

```suggestion
			return duration.Duration < duration2.Duration
```

Reply:
Good catch! I had changed `IsLess` to `IsLarger` at some point to make it more "obvious" the structure keeps the largest elements, but then forgot to change it back here 👍